### PR TITLE
chore: rename usdPrice -> priceUsd

### PR DESCRIPTION
**Pull request by jeanregisser**
### Description

Proposing to rename `usdPrice` -> `priceUsd` (+ some other related fields) to converge on the naming everywhere.
I used `priceUsd` in the [hooks](https://github.com/valora-inc/hooks/blob/f55b8090132776b67c4efc2cb50871cccd96b3a2/src/types/positions.ts#L103) repo and since we're introducing a new endpoint for [tokensInfo](https://github.com/valora-inc/wallet/pull/4210), now is the time to use a consistent name.

We can make it here without breaking older clients now.
Changing it in hooks is not possible without breaking backward compatibility.

To be merged once https://github.com/valora-inc/blockchain-api/pull/331 is merged.

Builds on top of https://github.com/valora-inc/wallet/pull/4210 where the new `/tokensInfo` was used.

### Test plan

- CI passes

### Related issues

- https://github.com/valora-inc/blockchain-api/pull/327#discussion_r1332797325

### Backwards compatibility

Yes

### CODE REVIEW

**Review by jh2oman — APPROVED** [review]

**Review by jophish — APPROVED** [review]
LGTM -- my branch is gonna be a pretty huge merge into main though!

**Comment by jeanregisser** [comment]
> LGTM -- my branch is gonna be a pretty huge merge into main though!

yes, I'll wait for your branch to be merged and then we can directly merge this one after?
this way it's more explicit.

What do you think @jophish?

**Pull request merged**
